src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.OpenTextBufferManager.cs

```diff
@@ -196,9 +196,11 @@ internal void SetReferenceSpans(IEnumerable<TextSpan> spans)
                         _referenceSpanToLinkedRenameSpanMap[span] = trackingSpan;
                     }
 
+                    UpdateReferenceSpanMappingCheck();
```

Comment:
This method checks the contents of the `_referenceSpanToLinkedRenameSpanMap` variable so I call this method any time the map is updated.